src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -41,23 +43,38 @@ object WorkflowActor {
 
   val AkkaTimeout = 5 seconds
 
-  type ExecutionStore = Map[Call, ExecutionStatus.Value]
+  type ExecutionStore = Map[ExecutionStoreKey, ExecutionStatus]
+  type ExecutionStoreEntry = (ExecutionStoreKey, ExecutionStatus)
+
+  // See also `WorkflowActor.createStore` that is similar, but uses the database.
+  // Currently only used for testing. May be merged / replaced with createStore.
```

Comment:
I'd like to have words like "basically" and "probably" fleshed out a bit more but if it's really not being used except for tests it shouldn't exist.